Cubical/Categories/UnderlyingGraph.agda

```diff
@@ -0,0 +1,54 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.UnderlyingGraph where
+
+open import Cubical.Categories.Morphism
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation hiding (_⟦_⟧)
+open import Cubical.Data.Graph.Base
+open import Cubical.Data.Sigma
+open import Cubical.Categories.Isomorphism
```

Comment:
We use the (undocumented - sorry) convention to start with the most basic imports like "Foundations" and group imports. In this case:
```
open import Cubical.Foundations.Prelude
open import Cubical.Foundations.HLevels

open import Cubical.Data.Graph.Base
open import Cubical.Data.Sigma

open import Cubical.Categories.Category.Base
open import Cubical.Categories.Functor
open import Cubical.Categories.Morphism
open import Cubical.Categories.NaturalTransformation hiding (_⟦_⟧)
open import Cubical.Categories.Isomorphism
```

Reply:
Is there some particular order? Like Foundations then Data/HITs then Categories?

Reply:
There is no fixed standard, but most of us use the lose convention that things get more specific from top to bottom. Which means "Foundations then Data/HITs then Categories" in this case.